commands/playwright-keplr.js

```diff
@@ -77,6 +79,10 @@ module.exports = {
     return keplrNotificationWindow;
   },
 
+  async clickByText(text, page = keplrWindow) {
+    await page.click(`text="${text}"`);
+  },
+
```

Comment:
how is this different from `waitAndClickByText`?

Reply:
`waitAndClickByText` works correctly when going via the create new wallet flow. It doesn't grab the button in the import existing wallet flow even though selectors are the same in both flows. `clickByText` is a utility I've created which gives consistent behavior for both, with no issues.